agent/config/const_windows.go

```diff
@@ -17,4 +17,12 @@
 package config
 
 // OSType is the type of operating system where agent is running
-const OSType = "windows"
+const (
+	OSType = "windows"
+	// containerAdminUser is the admin username for any container on Windows.
+	ContainerAdminUser = "ContainerAdministrator"
+	// This is the path that will be used to store the local named pipe for CSI Proxy
+	ManagedDaemonSocketPathHostRoot = "C:\\ProgramData\\Amazon\\ECS\\ebs-csi-driver"
```

Comment:
Same as comment from previous review. 

Please do not hardcode `ProgramData` path. It can change dynamically on ECS EC2.
Correct approach is to construct it via `ProgramData` Env as has been done in other parts of the agent.